charts/v5.0/cray-nls/Chart.yaml

```diff
@@ -24,18 +24,18 @@
 
 apiVersion: v2
 name: "cray-nls"
-version: 5.0.1  # Also update cray-iuf version and cray-nls.compatibility.yaml file
+version: 5.0.2  # Also update cray-iuf version and cray-nls.compatibility.yaml file
```

Comment:
Should this be at least a minor version update, instead of just a patch?

Reply:
Yes, I have changed it to be a minor version update.